src/UI/class-recommended-widget.php

```diff
@@ -14,6 +14,7 @@
 namespace Parsely\UI;
 
 use Parsely\Parsely;
+use Parsely\Recommended_Content;
```

Comment:
The changes on the recommended widget look good, but could we introduce them in a separate PR after this one gets merged?